lib/log.go

```diff
@@ -62,33 +64,48 @@ var (
 
 // LoggerConfig holds configuration settings for the logger, including logging level and output writer
 type LoggerConfig struct {
-	Level int32 `json:"level"`
-	Out   io.Writer
+	Level      int32 `json:"level"`         // logging level
+	Structured bool  `json:"logStructured"` // output structured logging for observability tooling
```

Comment:
Add in the default config function this structured logging as true (think it's the safe bet)

Reply:
not sure to be honest, for a deployment environment yes but for people who are going to run this for debugging purposes the non structured option could be more useful